<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmAddDelivery</class>
 <widget class="QWidget" name="frmAddDelivery">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="chkMediaReceived">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Media Received</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMediaType">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Media type:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblStaffing">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Staffing Level:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLocation">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Location:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboShipping">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Shipping</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hand-carried</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spnNumberObjects">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>50000</number>
   </property>
  </widget>
  <widget class="QComboBox" name="cboLocation">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Area 51 Nevada</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>King's Bay Georgia</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Great Lakes Illinois</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pensacola Florida</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bremerton Washington</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>New London Connecticut</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rota Spain</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblClassification">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Classification:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="dteDeliveryDate">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd MMM yyyy</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboStaffing">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cboMediaType">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Documentation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Software</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hardware</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cboClassification">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Unclassified</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Confidential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Secret</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblShipping">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Transit Method:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblDeliveryDate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Required Delivery Date:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNumberItems">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Items:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cboLocation</tabstop>
  <tabstop>cboClassification</tabstop>
  <tabstop>cboMediaType</tabstop>
  <tabstop>cboShipping</tabstop>
  <tabstop>spnNumberObjects</tabstop>
  <tabstop>cboStaffing</tabstop>
  <tabstop>dteDeliveryDate</tabstop>
  <tabstop>chkMediaReceived</tabstop>
  <tabstop>btnSubmit</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
